src/core/ext/filters/rbac/rbac_filter.cc

```diff
@@ -125,6 +125,7 @@ const grpc_channel_filter RbacFilter::kFilterVtable = {
 RbacFilter::RbacFilter(size_t index,
                        EvaluateArgs::PerChannelArgs per_channel_evaluate_args)
     : index_(index),
+      service_config_parser_index_(RbacServiceConfigParser::ParserIndex()),
```

Comment:
Need to change line 79 to actually use this field.

Reply:
done